capstone/src/components/User/Adventurer/Adventurer.jsx

```diff
@@ -1,28 +1,32 @@
 import './Adventurer.css';
 import { useNavigate, useParams } from 'react-router-dom';
-import { useEffect, useState, useMemo } from 'react';
+import { useEffect, useState, useMemo, useContext } from 'react';
 import axios from 'axios';
 import FindAdventure from './FindAdventure/FindAdventure';
 import Preference from './Preference/Preference';
 import Header from '../Header/Header';
 import ExperienceInfo from './ExperienceInfo/ExperienceInfo';
+import UserContext from '../../../Contexts/UserContext';
 
 // CONTEXTS
 import AdventurerContext from '../../../Contexts/AdventurerContext';
 
 const baseURL = process.env.REACT_APP_BASE_URL;
 
-function getNearbyRestaurants(location) {
-  return axios.post(`${baseURL}/adventure/restaurants`, location);
+function getNearbyRestaurants(lat, lng, username) {
+  const values = { lat, lng, username };
+  return axios.post(`${baseURL}/adventure/restaurants`, values);
```

Comment:
Does `values` need to be a separate variable? It'd be simpler to just inline it
```suggestion
  return axios.post(`${baseURL}/adventure/restaurants`, { lat, lng, username });
```